node.go

```diff
@@ -246,6 +250,30 @@ func (n *node) collectedCount() (count int64) {
 	return
 }
 
+func (n *node) emittedCount() (count int64) {
+	for _, out := range n.outs {
+		count += out.emittedCount()
```

Comment:
Hmm, to comment is gone, what it said was:

`node collected count is the sum of emitted counts of parent edges`

The opposite is true for node emitted count:

`node emitted count is the sum of collected counts of children edges`

Edges can buffer so we care about the actual points this node has collected or emitted.